DFE-Digital/claim-additional-payments-for-teaching
# [CAPT-721] Faster docker build

## What
Improve build time by optimising docker caching. A new pipeline is added to refresh the images regularly.

## How to review
- See improved build time in build pipeline: https://dev.azure.com/dfe-ssp/S118-Teacher-Payments-Service/_build/results?buildId=296362&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=68dc271a-6b90-554f-dc73-9843e30016f8
- Check refresh pipeline: https://dev.azure.com/dfe-ssp/S118-Teacher-Payments-Service/_build/results?buildId=296291&view=results
